compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1162,31 +1215,23 @@ class Allocator {
       Header->State = Chunk::State::Available;
     else
       Header->State = Chunk::State::Quarantined;
-    Header->OriginOrWasZeroed = useMemoryTagging<Config>(Options) &&
-                                Header->ClassId &&
-                                !TSDRegistry.getDisableMemInit();
-    Chunk::storeHeader(Cookie, Ptr, Header);
 
+    void *BlockBegin;
     if (UNLIKELY(useMemoryTagging<Config>(Options))) {
```

Comment:
Would it make sense to reverse the if and else clauses and check LIKELY(!useMemoryTagging...)?

It's a bit easier to read if the likely choice is first.